cluster/src/fpindexcluster/proxy_service.py

```
@@ -0,0 +1,267 @@
+        await runner.setup()
+        site = web.TCPSite(runner, self.config.proxy_host, self.config.proxy_port)
+        await site.start()
+        logger.info("Proxy service started")
+    
+    async def stop(self):
+        """Stop the proxy service"""
+        if self.nc:
+            await self.nc.close()
+        logger.info("Proxy service stopped")
+    
+    async def _publish_to_nats(self, index_name: str, fp_id: str, operation: str, data: Dict[str, Any]):
```

Ruff (0.12.2)

Missing return type annotation for private function `_publish_to_nats`

Add return type annotation: `None`

(ANN202)